readme.md

```diff
@@ -9,13 +9,34 @@ Using CRDTs to build a collaborative text editor, is mostly a solved problem. Tu
 
 [Repo](https://github.com/chewnoill/collaborative-editor)
 
+## Prerequisites
+
+1. node
+2. docker
+3. postgres
+4. dbmate (Installation: https://github.com/amacneil/dbmate#installation)
+
 ## Getting started
 
-Install dependencies:
+### Install dependencies:
 ```shell
 yarn
 ```
 
+### Docker setup:
+
+build
+```shell
+docker build && docker-compose build
```

Comment:
`docker-compose build` and `docker build` are redundant, you don't need to do both.